packages/components/addon/components/hds/link/inline.hbs

```diff
@@ -0,0 +1,23 @@
+{{! IMPORTANT: we removed the newlines before/after the yield to reduce the issues with unexpected whitespaces (see https://github.com/hashicorp/design-system/pull/231#issuecomment-1123502499) }}
+{{! IMPORTANT: we need to add "squishies" here (~) because otherwise the whitespace added by Ember becomes visible in the link (being an inline element) - See https://handlebarsjs.com/guide/expressions.html#whitespace-control }}
```

Comment:
I personally find whitespace control operators rather common in templates that handle non-block elements and whitespace-sensitive contexts; that being said it may be valuable for some people to have these comments in.

Reply:
I didn't know of their existence until a couple of weeks ago :)